index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+     <head lang="en">
+     <meta charset="UTF-8">
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+          <title> Travelmate</title>
+     </head>
+     <body>
+     <h1>
```

Comment:
Don't forget to close your `h1`!

```suggestion
     <h1></h1>
```